configuration/analyzer_config.json

```diff
@@ -258,6 +258,39 @@
     "secrets": {},
     "params": {}
   },
+  "CapeSandbox" : {
+    "type" : "file",
+    "python_module" : "cape_sandbox.CAPEsandbox",
+    "description" : "Automatic scan of suspicious files using CapeSandbox API",
+    "disabled" : false,
+    "external_service" : true,
+    "leaks_info" : true,
+    "docker_based" : false,
+    "config" : {
+      "soft_time_limit" : 500,
+      "queue" : "long"
+    },
+    "secrets" : {
+      "token" : {
+        "env_var_key" : "CAPESANDBOX_KEY",
+        "description" : "",
+        "required" : true
+      },
+    "url_key_name": {
```

Comment:
Install a JSON formatter in your IDE, the indentation here is messed up.

Reply:
Gotcha